api_app/schemas/shared_service.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/microsoft/AzureTRE/schema/shared_service.json",
+  "type": "object",
+  "title": "Shared Service Default Parameters",
+  "description": "These parameters are required for all shared services",
+  "required": [
+      "display_name",
+      "description"
+  ],
+  "properties": {
+      "display_name": {
+          "type": "string",
+          "title": "Name for the shared service",
+          "description": "The name of the shared service to be displayed to users"
```

Comment:
Don't worry about for this PR - but we could probably add `updateable: true` for display name and description.

Reply:
Added a note for my next PR!